atomos.maven/src/main/java/org/apache/felix/atomos/maven/AtomosIndexMojo.java

```diff
@@ -63,7 +63,7 @@ public void execute() throws MojoExecutionException
                 AtomosIndexMojo::isJarFile).map(a -> a.getFile().toPath()).collect(
                     Collectors.toList());
 
-            SubstrateUtil.indexContent(paths, outputDirectory.toPath());
+            //            SubstrateUtil.indexContent(paths, outputDirectory.toPath());
```

Comment:
I assume this still needs to be reworked to the refactored stuff.  I have on my list of things to do is add sniff integration tests to make sure the examples still work.  The atomos.examples.index resulting JAR no longer works because the atomos index is no longer written.